src/ExecuteTimeWidget.ts

```diff
@@ -131,7 +132,9 @@ export default class ExecuteTimeWidget extends Widget {
       );
       if (!executionTimeNode) {
         executionTimeNode = document.createElement('div') as HTMLDivElement;
-        executionTimeNode.className = EXECUTE_TIME_CLASS;
+        const displayClass =
```

Comment:
Can we make this a switch statement? Something like this?

```
let positioning;
switch(this._settings.display) {
case 'right':
    positioning = 'right';
    break;
default:
    positioning= 'left';
}

const displayClass =`${EXECUTE_TIME_CLASS}-display-${positioning }`;
```

Reply:
I've added the 'left' case explicitly and, an error in console if the positioning is unknown. Because `enum` is specified in the schema, this should never happen.